lfs/config.go

```diff
@@ -134,11 +141,10 @@ func (c *Configuration) BatchTransfer() bool {
 		}
 
 		// Any numeric value except 0 is considered true
-		if n, err := strconv.Atoi(v); err == nil && n != 0 {
-			return true
-		}
+		n, _ := strconv.Atoi(v)
+		return n != 0
```

Comment:
I'd just invert the whole thing (including removing the "true" test part) & return true unless you see "false" or successfully parse a zero. You're right that it will probably continue to work, I just don't really like relying on undocumented / non-guaranteed results.